test/fixtures/pass-output-style/webpack.config.js

```diff
@@ -24,7 +24,9 @@ module.exports = {
             {
               loader: loader,
               options: {
-                outputStyle: 'compressed'
+                sassOptions: {
+                  outputStyle: 'compressed'
+                }
```

Comment:
do not change test case of old options API, you should add a new case for sassOptions

Reply:
cf. comment above.